packages/unimodules-test-core/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "unimodules-test-core",
+  "version": "1.0.0",
+  "description": "UnimodulesTestCore standalone module",
+  "main": "build/Unimodule.js",
+  "types": "build/Unimodule.d.ts",
+  "scripts": {
+    "build": "expo-module build",
+    "clean": "expo-module clean",
+    "lint": "expo-module lint",
+    "test": "expo-module test",
+    "prepare": "expo-module prepare",
+    "prepublishOnly": "expo-module prepublishOnly",
+    "expo-module": "expo-module"
+  },
+  "keywords": [
+    "react-native",
+    "expo",
+    "unimodules-test-core"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo.git",
+    "directory": "packages/unimodules-test-core"
+  },
+  "bugs": {
+    "url": "https://github.com/expo/expo/issues"
+  },
+  "author": "650 Industries, Inc.",
+  "license": "MIT",
+  "homepage": "https://docs.expo.io/versions/latest/sdk/module-template",
+  "dependencies": {
+    "expo-module-scripts": "^1.2.0"
+  },
+  "jest": {
+    "preset": "expo-module-scripts/ios"
+  }
+}
```

File to be refactored and adapted as it should only provide minimal set of information that is required for this module. 
```suggestion
{
  "name": "unimodules-test-core",
  "version": "0.1.0",
  "description": "Module providing native testing utilities for testing unimodules",
  "keywords": [
    "react-native",
    "expo",
    "unimodules",
    "unimodules-test-core"
  ],
  "repository": {
    "type": "git",
    "url": "https://github.com/expo/expo.git",
    "directory": "packages/unimodules-test-core"
  },
  "bugs": {
    "url": "https://github.com/expo/expo/issues"
  },
  "author": "650 Industries, Inc.",
  "license": "MIT",
  "homepage": "https://github.com/expo/expo/tree/master/packages/unimodules-test-core"
}
```